src/domain/Game.java

```diff
@@ -9,12 +10,14 @@
 	private List<Player> players = new ArrayList<>();
 	private List<Card> deck = new ArrayList<>();
 	private int currentRound = 1;
+	private String currentPlayer;
+	private int amountOfPlayers;
```

Comment:
Dit is niet nodig aangezien we het aantal spelers gewoon uit de `players` lijst kunnen halen.